services/src/main/java/org/fao/geonet/api/records/MetadataApi.java

```diff
@@ -179,6 +182,10 @@ public String getRecord(
             required = true)
         @PathVariable
         String metadataUuid,
+        @Parameter(description = "Formatter to use on the record view page. " +
+            "If not specified, no redirect to the record view page is performed.")
```

Comment:
May also want to spec that invalid values are also ignored.
```suggestion
            "If invalid or not specified, no redirect to the record view page is performed.")
```